### ISSUE TITLE
Management fee introduction on Premier (PFR198)

### ISSUE THREAD

**Issue by Hubmar** [ovh/hosted-private-cloud-roadmap#81]
As you are aware, the management VMs (e.g. your vCenter) are hosted outside of your infrastructure, so that you can use 100% of your hosts' capacity (CPU, RAM).

Currently, the costs associated with management VM resource usage are integrated into the price of the initial 2TB datastores. From next month onwards, we will separate the cost of storage from the cost of management VMs. This will not affect the total of the final bill.

In short

In £, ex. VAT | Before your bill dated 01/10/2022 (excluded) | After your bill dated 01/10/2022 (included)
-- | -- | --
Management fees Premier range | 0 | 122
Datastore 2020-2000gb-ssd-full-Pack | 141 | 80
Datastore 2020-2000gb-ssd-full-Pack | 141 | 80
Total | 282 | 282
Option to delete initial datastores | No | Yes
